jscomp/core/js_packages_info.ml

```diff
@@ -238,7 +238,9 @@ let string_of_module_id
           else  
             begin match module_system with 
               | AmdJS | NodeJS | Es6 -> 
-                dep_package_name // dep_path // js_file
+                (* HACK - TODO Replace bs-stdlib upstream and not here *)
+                let upd_dep_package_name = if dep_package_name = "bs-platform" then "bs-stdlib" else dep_package_name in
```

Comment:
@bobzhang I tried hard to come up with a better solution than this, but I wasn't able to. Happy to take any pointers to where or how this replacement should be done.